package.json

```diff
@@ -38,10 +38,12 @@
     "@react-native-community/async-storage": "^1.9.0",
     "@react-native-community/geolocation": "^2.0.2",
     "@react-native-community/netinfo": "^5.5.0",
+    "@react-native-community/progress-bar-android": "^1.0.3",
+    "@react-native-community/progress-view": "^1.1.1",
```

Comment:
Where do we need these?

Reply:
They are required peer dependencies

Reply:
This has to be done according to react-native-pdf